types/assertions.d.ts

```diff
@@ -12,7 +12,7 @@ export type ThrowsExpectation = {
 	is?: Error;
 
 	/** The thrown error must have a message that equals the given string, or matches the regular expression. */
-	message?: string | RegExp;
+	message?: string | RegExp | Function;
```

Comment:
The type should be more specific than just `Function`.

Reply:
You are right. I didn't pay attention the first time. I have updated it now.